src/main/java/com/example/videotecha/dto/MovieDto.java

```diff
@@ -0,0 +1,74 @@
+package com.example.videotecha.dto;
+
+import com.example.videotecha.model.enums.Genre;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class MovieDto {
+
+    private String name;
+
+    private String director;
+
+    private List<Genre> genres = new ArrayList<>();
```

Comment:
No need to pre-init list here if you use an `@AllArgs` constructor

Reply:
We should not use lombok